# Icelandic translation for sbackup
# Copyright (c) 2010 Rosetta Contributors and Canonical Ltd 2010
# This file is distributed under the same license as the sbackup package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: sbackup\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-02-04 21:54+0100\n"
"PO-Revision-Date: 2010-09-24 21:40+0000\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: Icelandic <is@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2010-09-29 05:44+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#: src/sbackup/backupproc.py:169 src/sbackup/pkginfo.py:68
#, python-format
msgid "SBackup Daemon <%(login)s@%(hostname)s>"
msgstr ""

#: src/sbackup/backupproc.py:173
#, python-format
msgid "[SBackup] [%(profile)s] Report of %(date)s"
msgstr ""

#: src/sbackup/backupproc.py:178 src/sbackup/core/profile_handler.py:278
msgid "No log file specified."
msgstr ""

#: src/sbackup/backupproc.py:183
msgid "Unable to find log file."
msgstr ""

#: src/sbackup/backupproc.py:251
msgid ""
"Critical Error: No configuration file for the default profile was found!\n"
"\n"
"Now continue processing remaining profiles."
msgstr ""

#: src/sbackup/backupproc.py:300
msgid "Backup was canceled by user."
msgstr ""

#: src/sbackup/backupproc.py:305
msgid "An error occurred during the cancellation of backup:"
msgstr ""

#: src/sbackup/backupproc.py:311
msgid "An error occurred during the backup:"
msgstr ""

#: src/sbackup/backupproc.py:328
msgid "Error when sending email:"
msgstr ""

#: src/sbackup/backupproc.py:359
msgid "The following error(s) occurred before:"
msgstr ""

#: src/sbackup/backupproc.py:530
#, python-format
msgid ""
"Backup is not being started.\n"
"%s"
msgstr ""

#: src/sbackup/pkginfo.py:40
msgid ""
"This is a user friendly backup solution for common desktop needs. If you "
"want to help by submitting bugs, by translating or anything else please "
"visit our website."
msgstr ""

#: src/sbackup/pkginfo.py:43
msgid "Maintainers:"
msgstr ""

#: src/sbackup/pkginfo.py:47
msgid "Former maintainers:"
msgstr ""

#: src/sbackup/pkginfo.py:51
msgid "Contributors:"
msgstr ""

#: src/sbackup/pkginfo.py:56
msgid "translator-credits"
msgstr ""

#: src/sbackup/ar_backend/tar.py:109 src/sbackup/ar_backend/tar.py:167
#: src/sbackup/ar_backend/tar.py:213
msgid "Invalid archive type."
msgstr ""

#: src/sbackup/ar_backend/tar.py:123
#, python-format
msgid "Effective path for `%s` is not available"
msgstr ""

#: src/sbackup/ar_backend/tar.py:294
#, python-format
msgid "Unable to get effective path for `%s`"
msgstr ""

#: src/sbackup/ar_backend/tar.py:338
msgid ""
"For the moment split functionality is not compatible with compress option."
msgstr ""

#: src/sbackup/ar_backend/tar.py:347 src/sbackup/ar_backend/tar.py:408
#: src/sbackup/core/profile_handler.py:288
#, python-format
msgid "Unable to change permissions for file '%s'."
msgstr ""

#: src/sbackup/ar_backend/tar.py:356
#, python-format
msgid "Unable to remove temporary file `%(file)s`: %(error)s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:372
msgid "Launching TAR to make incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:401
msgid "Unable to find the SNAR file to make an incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:402
msgid "Falling back to full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:444
msgid "Launching TAR to make a full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:511
msgid "Leading '/' from member names were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:513
msgid "Leading '/' from hard link targets were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:522
#, python-format
msgid "TAR returned a message: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:524
#, python-format
msgid ""
"TAR returned messages:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:527
msgid "TAR has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:531
#, python-format
msgid "TAR returned a warning: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:533
#, python-format
msgid ""
"TAR returned warnings:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:536
msgid "TAR returned warnings but has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:540
msgid "Unable to finish successfully. TAR terminated with errors."
msgstr ""

#: src/sbackup/ar_backend/tar.py:721
#, python-format
msgid "Unable to terminate backend process: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:754
msgid "Included"
msgstr ""

#: src/sbackup/ar_backend/tar.py:754
msgid "Not changed"
msgstr ""

#: src/sbackup/ar_backend/tar.py:754
msgid "Directory"
msgstr ""

#: src/sbackup/ar_backend/tar.py:782
msgid "Line must be a string"
msgstr ""

#: src/sbackup/ar_backend/tar.py:798
msgid "Dumpdir inconsistancy : 'filename' is empty"
msgstr ""

#: src/sbackup/ar_backend/tar.py:811
msgid "Dumpdir inconsistancy : 'control' is empty"
msgstr ""

#: src/sbackup/ar_backend/tar.py:884
#, python-format
msgid "File '%s' does not exist."
msgstr ""

#: src/sbackup/ar_backend/tar.py:1014 src/sbackup/ar_backend/tar.py:1048
#: src/sbackup/ar_backend/tar.py:1148
msgid "The snarfile header is incomplete."
msgstr ""

#: src/sbackup/ar_backend/tar.py:1317
msgid "A SnapshotFile is required"
msgstr ""

#: src/sbackup/ar_backend/tar.py:1417
#, python-format
msgid "Directory does not exist: %s."
msgstr ""

#: src/sbackup/core/ConfigManager.py:330
#, python-format
msgid "Profile settings are being read from file '%s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:333
#, python-format
msgid ""
"Profile settings are being set to default values. Configuration file is set "
"to '%s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:479
msgid "Option 'target' is missing, aborting."
msgstr ""

#: src/sbackup/core/ConfigManager.py:547
#, python-format
msgid ""
"Error while processing configuration:\n"
"\n"
"%(error)s in line `%(line)s`.\n"
"\n"
"Please check your configuration file."
msgstr ""

#: src/sbackup/core/ConfigManager.py:588 src/sbackup/core/ConfigManager.py:622
#, python-format
msgid ""
"Unable to evaluate '%(parameter)s' as a dictionary (value got = '%(value)r')."
msgstr ""

#: src/sbackup/core/ConfigManager.py:614
#, python-format
msgid ""
"Couldn't evaluate '%(parameter)s' as a dictionary (value got = '%(value)r' )"
msgstr ""

#: src/sbackup/core/ConfigManager.py:638
msgid "You must provide a dictionary."
msgstr ""

#: src/sbackup/core/ConfigManager.py:755
#, python-format
msgid "Log output for [%(profile)s] is directed to file '%(file)s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:762
#, python-format
msgid "Log output for [%s] is not directed into a file."
msgstr ""

#: src/sbackup/core/ConfigManager.py:781
#, python-format
msgid "Unable to rename log file '%(src)s'->'%(dst)s': %(err)s"
msgstr ""

#: src/sbackup/core/ConfigManager.py:801
#, python-format
msgid "The config file '%s' couldn't be read!"
msgstr ""

#: src/sbackup/core/ConfigManager.py:813
#, python-format
msgid "section [%(section)s] in '%(configfile)s' should not exist, aborting"
msgstr ""

#: src/sbackup/core/ConfigManager.py:895
msgid "No schedule defined in configuration file. Probing from filesystem."
msgstr ""

#: src/sbackup/core/ConfigManager.py:988
#: src/sbackup/core/ConfigManager.py:1000
msgid "The config file is not set yet into this ConfigManager"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1163
msgid "No receiver set."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1165
msgid "No SMTP server set."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1168
msgid "Username set but no password specified."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1171
msgid "Password set but no username specified."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1174
msgid ""
"A certificate and key file is given while SSL option (smtptls=1) is not "
"set.\n"
"Select SSL in order to use Certificate and Key."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1178
msgid ""
"When specifying a SSL certificate or key file, a key file resp. certificate "
"is mandatory."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1369
msgid "Default Profile"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1370
msgid "Unknown Profile"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1423
msgid "Unlimited"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1424
msgid "100 MiB"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1425
msgid "250 MiB"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1426
msgid "650 MiB"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1427
msgid "2 GiB (FAT16)"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1428
msgid "4 GiB (FAT32)"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1429 src/sbackup/ui/configgui.py:279
#: src/sbackup/ui/configgui.py:1418
msgid "Custom"
msgstr "Sérniðið"

#: src/sbackup/core/ConfigManager.py:1431
msgid "MP3 Music"
msgstr "MP3 Tónlist"

#: src/sbackup/core/ConfigManager.py:1432
msgid "AVI Video"
msgstr "AVI Video"

#: src/sbackup/core/ConfigManager.py:1433
#: src/sbackup/core/ConfigManager.py:1434
msgid "MPEG Video"
msgstr "MPEG Video"

#: src/sbackup/core/ConfigManager.py:1435
msgid "Matrjoshka Video"
msgstr "Matrjoshka Video"

#: src/sbackup/core/ConfigManager.py:1436
msgid "OGG Multimedia container"
msgstr "OGG Margmiðlunar skrár"

#: src/sbackup/core/ConfigManager.py:1437
msgid "CD Images"
msgstr "CD myndir"

#: src/sbackup/core/RestoreManager.py:80
#, python-format
msgid "File '%s' not found in the backup snapshot files list"
msgstr ""

#: src/sbackup/core/RestoreManager.py:82
#, python-format
msgid ""
"File '%(filename)s' not found in snapshot's [%(snapshotname)s] files list, "
"Skipped."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:119
#, python-format
msgid "Snapshot '%s' not found "
msgstr ""

#: src/sbackup/core/SnapshotManager.py:213
#, python-format
msgid "Invalid snapshot `%(name)s` found: Name of snapshot not valid."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:216
#, python-format
msgid "Invalid snapshot `%(name)s` found: %(error_cause)s."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:305
#, python-format
msgid "No need to rebase a full snapshot '%s'"
msgstr ""

#: src/sbackup/core/SnapshotManager.py:307
#, python-format
msgid ""
"'%(snapshotToRebase)s'  doesn't have 'base' file , it might have been broken "
msgstr ""

#: src/sbackup/core/SnapshotManager.py:312
#, python-format
msgid ""
"Cannot rebase a snapshot on an earlier one : '%(snapshotToRebase)s' <= '%"
"(NewBaseSnapshot)s' "
msgstr ""

#: src/sbackup/core/SnapshotManager.py:319
#, python-format
msgid "Rebasing '%(current)s' on '%(base)s'"
msgstr ""

#: src/sbackup/core/SnapshotManager.py:556
#, python-format
msgid "An error occurred when pulling snapshot '%s'."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:595
msgid "The snarfile header is incomplete !"
msgstr ""

#: src/sbackup/core/SnapshotManager.py:747
#, python-format
msgid "Base of snapshot '%s' is not set."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:784
#, python-format
msgid "Snapshot '%s' is already Full, nothing to do (not changing it to full)."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:824
#, python-format
msgid "Cancelling pull of snapshot '%s'"
msgstr ""

#: src/sbackup/core/UpgradeManager.py:176
#, python-format
msgid "Unable to remove upgrade stamp: %s"
msgstr ""

#: src/sbackup/core/UpgradeManager.py:183
#, python-format
msgid "Upgrading from v1.0 to v1.2: %s"
msgstr ""

#: src/sbackup/core/UpgradeManager.py:202
#, python-format
msgid "Upgrading to v1.3: %s"
msgstr ""

#: src/sbackup/core/UpgradeManager.py:229
#, python-format
msgid "Upgrading to v1.4: %s"
msgstr ""

#: src/sbackup/core/UpgradeManager.py:246
#, python-format
msgid "Upgrading to v1.5: %s"
msgstr ""

#: src/sbackup/core/UpgradeManager.py:300
#, python-format
msgid "The SNAR file already exist for snapshot '%s'. It is not overwritten."
msgstr ""

#: src/sbackup/core/profile_handler.py:90
msgid "Preparation of backup process"
msgstr ""

#: src/sbackup/core/profile_handler.py:133
msgid ""
"There are snapshots stored in outdated snapshot formats. Please upgrade them "
"using 'Simple Backup-Restoration' if you want to use them."
msgstr ""

#: src/sbackup/core/profile_handler.py:142
#, python-format
msgid "Snapshot '%(name)s' is being made."
msgstr ""

#: src/sbackup/core/profile_handler.py:150
#, python-format
msgid "Setting Base to '%(value)s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:159
msgid "Setting packages File."
msgstr ""

#: src/sbackup/core/profile_handler.py:165
msgid "Problem when setting the packages list: "
msgstr ""

#: src/sbackup/core/profile_handler.py:169
msgid "Setting Excludes File."
msgstr ""

#: src/sbackup/core/profile_handler.py:177
#, python-format
msgid "Setting compression format to `%s`"
msgstr ""

#: src/sbackup/core/profile_handler.py:184
#, python-format
msgid "Setting size of archive chunks to %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:191
msgid "Option 'Follow symbolic links' is enabled."
msgstr ""

#: src/sbackup/core/profile_handler.py:193
msgid "Option 'Follow symbolic links' is disabled."
msgstr ""

#: src/sbackup/core/profile_handler.py:202
msgid "Snapshot is being committed"
msgstr ""

#: src/sbackup/core/profile_handler.py:216
#, python-format
msgid "Error while purging old snapshots: %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:218
msgid "Backup process finished."
msgstr ""

#: src/sbackup/core/profile_handler.py:225
msgid "Inspect file system and collect file infos"
msgstr ""

#: src/sbackup/core/profile_handler.py:236
msgid "Summary of backup"
msgstr ""

#: src/sbackup/core/profile_handler.py:237
#, python-format
msgid "Number of directories: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:238
#, python-format
msgid "Total number of files: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:239
#, python-format
msgid "Number of symlinks: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:240
#, python-format
msgid "Number of files included in snapshot: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:241
#, python-format
msgid "Number of new files (also included): %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:242
#, python-format
msgid "Number of files skipped in incremental snapshot: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:243
#, python-format
msgid "Number of items forced to be excluded: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:244
#, python-format
msgid "Number of items to be excluded by config: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:245
#, python-format
msgid "Maximum free size required is '%s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:251
#, python-format
msgid "Available disk size is '%s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:253
#, python-format
msgid ""
"Not enough free space in the target directory for the planned backup (free: %"
"(freespace)s, required: %(neededspace)s)."
msgstr ""

#: src/sbackup/core/profile_handler.py:273
msgid "Unable to copy log. File access is not initialized."
msgstr ""

#: src/sbackup/core/profile_handler.py:291
#, python-format
msgid "Unable to copy log file: %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:293
msgid "Unable to find logfile to copy into snapshot."
msgstr ""

#: src/sbackup/core/profile_handler.py:295
msgid "No snapshot to copy logfile."
msgstr ""

#: src/sbackup/core/profile_handler.py:302
msgid "Processing of profile was canceled on user request\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:319
msgid "Processing of profile successfully finished (no errors)\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:325
#, python-format
msgid "Processing of profile failed with error: %s\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:336
#, python-format
msgid "Backup destination: %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:340
msgid "Unable to find destination directory."
msgstr ""

#: src/sbackup/core/profile_handler.py:359
msgid "Unable to find destination directory even after retry."
msgstr ""

#: src/sbackup/core/profile_handler.py:360
#: src/sbackup/core/profile_handler.py:367
#, python-format
msgid "Target directory '%(target)s' does not exist."
msgstr ""

#: src/sbackup/core/profile_handler.py:373
#, python-format
msgid "Unable to access destination: %s"
msgstr ""

#: src/sbackup/core/snapshot.py:132
msgid "Snapshot is inconsistent: __name is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:143
#, python-format
msgid "Name of snapshot '%s' doesn't match requirement"
msgstr ""

#: src/sbackup/core/snapshot.py:237
msgid "Snapshot is inconsistent: __snapshotpath is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:324
#, python-format
msgid ""
"The snapshot compression format is supposed to be '%s' but the corresponding "
"well named file wasn't found"
msgstr ""

#: src/sbackup/core/snapshot.py:348
#, python-format
msgid ""
"%(file)s doesn't contain valid value. Ignoring incomplete or non - backup "
"directory. "
msgstr ""

#: src/sbackup/core/snapshot.py:484
#, python-format
msgid "Name of Snapshot not valid : %s"
msgstr ""

#: src/sbackup/core/snapshot.py:502
#, python-format
msgid "Name of base not valid : %s"
msgstr ""

#: src/sbackup/core/snapshot.py:548
#, python-format
msgid "Invalid name of snapshot: %s"
msgstr ""

#: src/sbackup/core/snapshot.py:552
msgid "Unable to read mandatory file `ver`"
msgstr ""

#: src/sbackup/core/snapshot.py:597
msgid "Base name must be set for incremental backup."
msgstr ""

#: src/sbackup/fs_backend/_fuse_fam.py:106
msgid "Initializing FUSE File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_fuse_fam.py:129
msgid "Terminating FUSE File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:44
#: src/sbackup/fs_backend/_gio_utils.py:107
msgid "SFTP"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:45
#: src/sbackup/fs_backend/_gio_utils.py:108
msgid "SSH"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:46
#: src/sbackup/fs_backend/_gio_utils.py:109
msgid "FTP"
msgstr ""

#: src/sbackup/fs_backend/_gio_fam.py:100
msgid "Initializing GIO File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_gio_fam.py:114
msgid "Terminating GIO File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:110
msgid "NFS"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:111
msgid "SMB"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:191
msgid "Unable to get mount: path is not mounted"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:193
msgid "Unable to get mount: path not found when mounting (is probably local)"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:195
#: src/sbackup/fs_backend/_gio_utils.py:198
#, python-format
msgid "Unable to get mount: %s"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:248
msgid "Unable to un-set mount flag: path is not stored"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:350
msgid "Path is already mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:378
msgid "Path is not mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:846
msgid "test specified path for existence using GIO"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:859
msgid "Create testdir"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:864
msgid "Test testfile for existence"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:871
msgid "Create file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:878
msgid "Re-read test file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:890
msgid "Remove file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:892
msgid "Remove dir"
msgstr ""

#: src/sbackup/fuse_plugins/__init__.py:160
#, python-format
msgid "Unable to import plugin `%(plugin_name)s`: %(error_cause)s "
msgstr ""

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:117
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of ftp "
"sites: %s"
msgstr ""

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:127
#: src/sbackup/fuse_plugins/sftpFuseFAM.py:124
#, python-format
msgid "Couldn't mount '%(server)s' into '%(mountpoint)s' : %(error)s"
msgstr ""

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:141
msgid "FTP schema is: ftp://user:pass@server/anything"
msgstr ""

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:188
#, python-format
msgid ""
"Error matching the schema 'ftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:114
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of sftp "
"sites: %s"
msgstr ""

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:138
msgid "SFTP schema is like : sftp://user:pass@server/anything"
msgstr ""

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:185
#, python-format
msgid ""
"Error matching the schema 'sftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""

#: src/sbackup/fuse_plugins/sshFuseFAM.py:81
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com/home/' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""

#: src/sbackup/fuse_plugins/sshFuseFAM.py:137
#, python-format
msgid ""
"The sshfs command '%(command)s' didn't perform normally. Output => %"
"(erroroutput)s "
msgstr ""

#: src/sbackup/fuse_plugins/sshFuseFAM.py:142
msgid "SSH schema is like : ssh://user:pass@example.com:33/home/user/backup/"
msgstr ""

#: src/sbackup/fuse_plugins/sshFuseFAM.py:158
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com:21/home/' with '%"
"s' (The '/' after server is mandatory)"
msgstr ""

#: src/sbackup/ui/GladeWindow.py:210
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target."
msgstr ""

#: src/sbackup/ui/GladeWindow.py:211
#, python-format
msgid ""
"While accessing the backup destination following error occurred:\n"
"\n"
"%s."
msgstr ""

#: src/sbackup/ui/GladeWindow.py:212 src/sbackup/ui/restoregui.py:817
#: src/sbackup/ui/restoregui.py:871
msgid "Simple Backup error"
msgstr ""

#: src/sbackup/ui/GladeWindow.py:213
msgid "Unable to access backup destination"
msgstr ""

#: src/sbackup/ui/configgui.py:125
msgid "Scheduled backups are available for Administrator users only."
msgstr ""

#: src/sbackup/ui/configgui.py:135 src/sbackup/ui/configgui.py:144
msgid "Name"
msgstr ""

#: src/sbackup/ui/configgui.py:152
msgid "File Type"
msgstr ""

#: src/sbackup/ui/configgui.py:187
msgid "Enable"
msgstr ""

#: src/sbackup/ui/configgui.py:188
msgid "Profile Name"
msgstr ""

#: src/sbackup/ui/configgui.py:285
#, python-format
msgid ""
"Invalid or empty regular expression ('%s') found in configuration. Removed."
msgstr ""

#: src/sbackup/ui/configgui.py:292
#, python-format
msgid ""
"Invalid or empty regular expressions found\n"
"in configuration file:\n"
"'%s'\n"
"\n"
"These expressions are not used and were\n"
"removed from the configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:548
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target to an existing one."
msgstr ""

#: src/sbackup/ui/configgui.py:549
#, python-format
msgid ""
"Backup target `%s` does not exist.\n"
"\n"
"Attention: The target will be set to the default value. Check this on the "
"destination settings page before saving the configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:551
msgid "Unable to open backup target"
msgstr ""

#: src/sbackup/ui/configgui.py:623
#, python-format
msgid ""
"%s\n"
"\n"
"A backup profile using default values was created. Save the new "
"configuration in order to use it or check your existing configuration file "
"manually.\n"
"\n"
"Important note: Saving will overwrite your existing invalid configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:641
#, python-format
msgid "Current profile: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:732
msgid "Already included item !"
msgstr "Þessi skrá/mappa hefur þegar verið valin !"

#: src/sbackup/ui/configgui.py:747
msgid "Already excluded item !"
msgstr "Hef þegar útilokað þennan hlut"

#: src/sbackup/ui/configgui.py:759 src/sbackup/ui/configgui.py:1458
msgid "Empty expression. Please enter a valid regular expression."
msgstr "Tóm segð. Sláðu in gilda regexp segð."

#: src/sbackup/ui/configgui.py:774 src/sbackup/ui/configgui.py:1472
msgid "Provided regular expression is not valid."
msgstr ""

#: src/sbackup/ui/configgui.py:782
msgid "Empty filename or path. Please enter a valid filename or path."
msgstr ""
"Tómt skráarheiti eða tóm slóð. Sláðu inn gilt skráarheiti eða gilda slóð."

#: src/sbackup/ui/configgui.py:791
msgid ""
"It seems the path you entered does not exists. Do you want to add this "
"incorrect path?"
msgstr ""

#: src/sbackup/ui/configgui.py:820
msgid "Save configuration as..."
msgstr ""

#: src/sbackup/ui/configgui.py:887
msgid ""
"A backup process is now executed in the background.\n"
"\n"
"You can monitor the progress of the backup by means of the status indicator "
"displayed in the notification area."
msgstr ""

#: src/sbackup/ui/configgui.py:888
msgid "Backup process started"
msgstr ""

#: src/sbackup/ui/configgui.py:893 src/sbackup/ui/configgui.py:1661
#: src/sbackup/ui/configgui.py:2038
#, python-format
msgid "Unexpected dialog response: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:967
msgid "Include file..."
msgstr ""

#: src/sbackup/ui/configgui.py:974 src/sbackup/ui/configgui.py:1029
msgid "All files"
msgstr "Allar skrár"

#: src/sbackup/ui/configgui.py:994
msgid "Include folder..."
msgstr ""

#: src/sbackup/ui/configgui.py:1022
msgid "Exclude file..."
msgstr ""

#: src/sbackup/ui/configgui.py:1045
msgid "Exclude folder..."
msgstr ""

#: src/sbackup/ui/configgui.py:1310
msgid "Configuration has changed"
msgstr ""

#: src/sbackup/ui/configgui.py:1311
msgid ""
"There are unsaved modifications. Please save the configuration or revert "
"these changes before testing the mail settings."
msgstr ""

#: src/sbackup/ui/configgui.py:1317 src/sbackup/ui/configgui.py:1323
msgid "Test mail settings"
msgstr ""

#: src/sbackup/ui/configgui.py:1318
msgid "The test failed with following output:"
msgstr ""

#: src/sbackup/ui/configgui.py:1324
msgid "The test was successful."
msgstr ""

#: src/sbackup/ui/configgui.py:1407 src/sbackup/ui/configgui.py:1450
#, python-format
msgid ""
"The given expression contains unsupported characters ('%s'). Currently it is "
"not possible to use these characters in exclude expressions."
msgstr ""

#: src/sbackup/ui/configgui.py:1410 src/sbackup/ui/configgui.py:1453
msgid "Unsupported character"
msgstr ""

#: src/sbackup/ui/configgui.py:1621
msgid "Connect to remote destination"
msgstr ""

#: src/sbackup/ui/configgui.py:1669
msgid "Mount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1680
msgid "Unable to mount host"
msgstr ""

#: src/sbackup/ui/configgui.py:1689
msgid "Umount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1699
msgid "Unable to unmount host"
msgstr ""

#: src/sbackup/ui/configgui.py:1729
msgid "Perfom tests on remote host"
msgstr ""

#: src/sbackup/ui/configgui.py:1742
msgid "Unable to access remote destination"
msgstr ""

#: src/sbackup/ui/configgui.py:1746
msgid "All tests passed"
msgstr ""

#: src/sbackup/ui/configgui.py:1910
msgid ""
"The given name of the new profile is empty. Please enter a valid profile "
"name."
msgstr ""

#: src/sbackup/ui/configgui.py:1911 src/sbackup/ui/configgui.py:1918
msgid "Profile name not valid"
msgstr ""

#: src/sbackup/ui/configgui.py:1917
msgid ""
"The given name of the new profile already exists. Please enter another name."
msgstr ""

#: src/sbackup/ui/configgui.py:1919
msgid "Renaming of profiles is not supported."
msgstr ""

#: src/sbackup/ui/configgui.py:1949
#, python-format
msgid ""
"<b>Delete configuration profile?</b>\n"
"\n"
"You are trying to remove a configuration profile. You will not be able to "
"restore it. If you are not sure, use the 'enable|disable' functionality "
"instead.\n"
"\n"
"Do you really want to delete the profile '%(name)s'?"
msgstr ""

#: src/sbackup/ui/configgui.py:2022
#, python-format
msgid ""
"<big><b>Set default values for current profile?</b></big>\n"
"This will restore the default values for the profile currently edited: '%"
"s'.\n"
"\n"
"These predefined settings are recommended for most users. Check whether they "
"are appropriate for your use before saving the changed configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:2032
msgid "Default settings are being applied."
msgstr ""

#: src/sbackup/ui/configgui.py:2057
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be removed. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2071
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be disabled. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2085
msgid ""
"<b>No backup profile found.</b>\n"
"\n"
"No default profile was found. You are probably running Simple Backup for the "
"first time. A backup profile using default values was created.\n"
"\n"
"Please modify the settings according to your needs and save the "
"configuration in order to use it."
msgstr ""

#: src/sbackup/ui/configgui.py:2098
msgid "Please select a profile."
msgstr ""

#: src/sbackup/ui/indicator_core.py:117
msgid "there was a problem initializing the pynotify module"
msgstr ""

#: src/sbackup/ui/indicator_core.py:146 src/sbackup/ui/indicator_core.py:193
#: src/sbackup/ui/indicator_core.py:221 src/sbackup/ui/indicator_core.py:243
#, python-format
msgid "Connection to notification-daemon failed: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:306
msgid "Simple Backup"
msgstr ""

#: src/sbackup/ui/indicator_core.py:312
msgid "Show message windows"
msgstr ""

#: src/sbackup/ui/indicator_core.py:354 src/sbackup/ui/indicator_core.py:814
msgid "Cancel Backup"
msgstr ""

#: src/sbackup/ui/indicator_core.py:416
msgid "Connection to D-Bus service lost."
msgstr ""

#: src/sbackup/ui/indicator_core.py:441
msgid "re-connection was successful"
msgstr ""

#: src/sbackup/ui/indicator_core.py:480
msgid ""
"No backup in progress and connection to D-Bus service lost. Simple Backup "
"Indicator is being terminated."
msgstr ""

#: src/sbackup/ui/indicator_core.py:509
msgid "Starting backup session"
msgstr ""

#: src/sbackup/ui/indicator_core.py:523
msgid "Ending Backup Session"
msgstr ""

#: src/sbackup/ui/indicator_core.py:536
msgid ""
"There are snapshots with old snapshot format. Please upgrade these using the "
"Restoration tool if you want to use them."
msgstr ""

#: src/sbackup/ui/indicator_core.py:538
#, python-format
msgid "Unknown D-Bus event `%s` received."
msgstr ""

#: src/sbackup/ui/indicator_core.py:579
msgid "Try again"
msgstr ""

#: src/sbackup/ui/indicator_core.py:649
msgid ""
"Attempt of starting another instance of Simple Backup while this one is "
"already running."
msgstr ""

#: src/sbackup/ui/indicator_core.py:659
#, python-format
msgid "Unable to present window: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:675
#, python-format
msgid "Do you really want to cancel the backup process of profile '%s'?"
msgstr ""

#: src/sbackup/ui/indicator_core.py:679
msgid "Cancel running backup process?"
msgstr ""

#: src/sbackup/ui/indicator_core.py:696
msgid "Unable to cancel backup process."
msgstr ""

#: src/sbackup/ui/indicator_core.py:758
#, python-format
msgid "Profile: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:759
#, python-format
msgid "Size of backup: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:760
#, python-format
msgid "%.1f%% processed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:761
#, python-format
msgid "Remaining time: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:807
#, python-format
msgid "PID of backup process being canceled: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:812
#, python-format
msgid "Unable to send signal to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:813
#, python-format
msgid "Sending signal using gksu to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:817
#, python-format
msgid "Sending signal using gksu to process %s failed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:852
#, python-format
msgid ""
"<b>Unable to find specified target directory</b>\n"
"\n"
"The specified target directory '%s' was not found.\n"
"\n"
msgstr ""

#: src/sbackup/ui/indicator_core.py:853
#, python-format
msgid ""
"You can try to use the specified target again or cancel the profile "
"execution. The specified destination is automatically used in %s seconds."
msgstr ""

#: src/sbackup/ui/indicator_core.py:880 src/sbackup/ui/indicator_core.py:922
#: src/sbackup/ui/indicator_core.py:940 src/sbackup/ui/indicator_core.py:948
#: src/sbackup/ui/indicator_core.py:965
msgid "unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:901
msgid "less than 1 minute"
msgstr ""

#: src/sbackup/ui/indicator_core.py:903
msgid "about 2 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:905
msgid "about 3 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:907
msgid "about 4 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:909
msgid "about 5 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:911
#, python-format
msgid "about %.0f minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:921
msgid "In progress"
msgstr ""

#: src/sbackup/ui/indicator_core.py:931 src/sbackup/ui/indicator_core.py:957
msgid "finished"
msgstr ""

#: src/sbackup/ui/indicator_core.py:939
msgid "Progress: unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:946
msgid "Backup is being prepared"
msgstr ""

#: src/sbackup/ui/indicator_core.py:947
msgid "Collecting file info"
msgstr ""

#: src/sbackup/ui/indicator_core.py:956
msgid "Progress: canceled"
msgstr ""

#: src/sbackup/ui/indicator_core.py:978
msgid "An error occurred"
msgstr ""

#: src/sbackup/ui/indicator_core.py:1011
msgid "Another `Simple Backup Indicator` is already running."
msgstr ""

#: src/sbackup/ui/indicator_core.py:1035
#, python-format
msgid "Initialization of GTK+ failed: %s"
msgstr ""

#: src/sbackup/ui/misc.py:294
#, python-format
msgid "%s   (Administrator mode)"
msgstr ""

#: src/sbackup/ui/misc.py:324
#, python-format
msgid ""
"An uncaught error occurred. Close this message window and restart the "
"application.\n"
"\n"
"Please report this error on https://bugs.launchpad.net/sbackup.\n"
"\n"
"%s"
msgstr ""

#: src/sbackup/ui/misc.py:325
msgid "Sorry, this should not have happened"
msgstr ""

#: src/sbackup/ui/restoregui.py:78
msgid "Restore..."
msgstr ""

#: src/sbackup/ui/restoregui.py:79 data/ui/sbackup-restore.glade:845
msgid "Restore as..."
msgstr ""

#: src/sbackup/ui/restoregui.py:80
msgid "Revert..."
msgstr ""

#: src/sbackup/ui/restoregui.py:81 data/ui/sbackup-restore.glade:877
msgid "Revert as..."
msgstr ""

#: src/sbackup/ui/restoregui.py:128 data/ui/sbackup-restore.glade:111
#: data/ui/sbackup-restore.glade:591
msgid "n.a."
msgstr ""

#: src/sbackup/ui/restoregui.py:146 src/sbackup/ui/restoregui.py:163
msgid "Snapshots"
msgstr ""

#: src/sbackup/ui/restoregui.py:147
msgid "Version"
msgstr ""

#: src/sbackup/ui/restoregui.py:154 data/ui/sbackup-config.glade:4063
msgid "Path"
msgstr "Slóð"

#: src/sbackup/ui/restoregui.py:159
msgid "State"
msgstr ""

#: src/sbackup/ui/restoregui.py:187
msgid "Connect to destination..."
msgstr ""

#: src/sbackup/ui/restoregui.py:211
msgid "Specified path does not exist"
msgstr ""

#: src/sbackup/ui/restoregui.py:257
msgid "Ready"
msgstr ""

#: src/sbackup/ui/restoregui.py:371
msgid "No destination specified"
msgstr ""

#: src/sbackup/ui/restoregui.py:388
msgid "Choose a source folder"
msgstr "Veldu áfangastað"

#: src/sbackup/ui/restoregui.py:405
msgid "Reading snapshots..."
msgstr ""

#: src/sbackup/ui/restoregui.py:428
msgid "Reading directory content..."
msgstr ""

#: src/sbackup/ui/restoregui.py:458 src/sbackup/ui/restoregui.py:496
msgid "Loading ..."
msgstr "Hleð ..."

#: src/sbackup/ui/restoregui.py:539
msgid "Reading files from snapshot..."
msgstr ""

#: src/sbackup/ui/restoregui.py:552
msgid "Simple Backup restore error"
msgstr ""

#: src/sbackup/ui/restoregui.py:553
msgid "An error occurred while reading snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:557
msgid "This snapshot seems empty."
msgstr ""

#: src/sbackup/ui/restoregui.py:576
msgid "Reading backup snapshot..."
msgstr ""

#: src/sbackup/ui/restoregui.py:599
msgid "No backups found for this day."
msgstr ""

#: src/sbackup/ui/restoregui.py:614
#, python-format
msgid ""
"Snapshot version is not supported (Only %(supportedversion)s is supported). "
"Version '%(currentversion)s' found. You should upgrade it. "
msgstr ""

#: src/sbackup/ui/restoregui.py:643
msgid "Select restore location"
msgstr "Veldu staðsetningu fyrir endurheimtu gögnin"

#: src/sbackup/ui/restoregui.py:662
#, python-format
msgid "Do you really want to revert '%s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:676
msgid "Select revert location"
msgstr ""

#: src/sbackup/ui/restoregui.py:685
#, python-format
msgid "Do you really want to revert '%(source)s' to '%(dir)s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:763
msgid "Reading snapshot..."
msgstr ""

#: src/sbackup/ui/restoregui.py:782
msgid "The version of the snapshot is greater than the supported one!"
msgstr ""

#: src/sbackup/ui/restoregui.py:792
#, python-format
msgid ""
"Upgrading the selected snapshot `%s` might corrupt your backup for this "
"particular point in time. You cannot undo this operation. It is recommended "
"to create a copy of the snapshot directory before upgrading it. The upgrade "
"process takes a while - be patient.\n"
"\n"
"Are you sure that you want to upgrade the snapshot now?"
msgstr ""

#: src/sbackup/ui/restoregui.py:796
msgid "Do you really want to upgrade snapshot?"
msgstr ""

#: src/sbackup/ui/restoregui.py:800
#, python-format
msgid "Upgrade snapshot '%s'"
msgstr ""

#: src/sbackup/ui/restoregui.py:809
msgid "Snapshot was sucessfully upgraded."
msgstr ""

#: src/sbackup/ui/restoregui.py:810
msgid "Simple Backup upgrade"
msgstr ""

#: src/sbackup/ui/restoregui.py:811
msgid "Upgrade finished"
msgstr ""

#: src/sbackup/ui/restoregui.py:814
#, python-format
msgid ""
"While attempting to upgrade snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:817
msgid "Unable to upgrade snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:838
#, python-format
msgid "Do you really want to rebase '%(current)s' on '%(base)s' ?"
msgstr ""

#: src/sbackup/ui/restoregui.py:856
#, python-format
msgid ""
"Deleting the selected snapshot `%s` will remove your backup for this "
"particular point in time. You cannot undo this operation.\n"
"\n"
"Are you sure that you want to remove the snapshot permanently?"
msgstr ""

#: src/sbackup/ui/restoregui.py:860
msgid "Do you really want to remove snapshot?"
msgstr ""

#: src/sbackup/ui/restoregui.py:869
#, python-format
msgid ""
"While attempting to delete snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:872
msgid "Unable to delete snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:878
#, python-format
msgid ""
"The selected snapshot '%s' is referenced by more recent snapshots as base "
"snapshot.\n"
"\n"
"Remove all of these child snapshots in order to delete this snapshot."
msgstr ""

#: src/sbackup/ui/restoregui.py:880
msgid "Unable to remove snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:888
msgid "Termination in progress"
msgstr ""

#: src/sbackup/ui/restoregui.py:904 src/sbackup/ui/restoregui.py:910
#: src/sbackup/ui/restoregui.py:916 src/sbackup/ui/restoregui.py:922
#: data/ui/sbackup-restore.glade:1039
msgid "Simple Backup restoration"
msgstr ""

#: src/sbackup/ui/restoregui.py:905 src/sbackup/ui/restoregui.py:911
msgid "<b>Restoring of selected files</b>"
msgstr ""

#: src/sbackup/ui/restoregui.py:906
#, python-format
msgid "Restoring of <tt>'%s'</tt> is in progress."
msgstr ""

#: src/sbackup/ui/restoregui.py:907
#, python-format
msgid "Restoring of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:908
#, python-format
msgid ""
"Restoring of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/ui/restoregui.py:912
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""

#: src/sbackup/ui/restoregui.py:913
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:914
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/ui/restoregui.py:917 src/sbackup/ui/restoregui.py:923
msgid "<b>Reverting selected files</b>"
msgstr ""

#: src/sbackup/ui/restoregui.py:918
#, python-format
msgid "Reverting of <tt>'%s'</tt> is in progress.\n"
msgstr ""

#: src/sbackup/ui/restoregui.py:919
#, python-format
msgid "Reverting of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:920
#, python-format
msgid ""
"Reverting of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/ui/restoregui.py:924
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""

#: src/sbackup/ui/restoregui.py:925
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:926
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""

#: src/sbackup/util/__init__.py:439
#, python-format
msgid "%(mb)d MiB %(kb)d KiB"
msgstr ""

#: src/sbackup/util/__init__.py:442
#, python-format
msgid "%(mb)d MiB %(kb)d KiB %(bytes)d"
msgstr ""

#: src/sbackup/util/__init__.py:447
#, python-format
msgid "%(mb)d MB %(kb)d kB"
msgstr ""

#: src/sbackup/util/__init__.py:450
#, python-format
msgid "%(mb)d MB %(kb)d kB %(bytes)d"
msgstr ""

#: src/sbackup/util/filecollect.py:335
#, python-format
msgid "File '%(file)s' does not exist."
msgstr ""

#: src/sbackup/util/filecollect.py:348
#, python-format
msgid "File '%(file)s' is not accessable with error '%(error)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:364
#, python-format
msgid "File '%(file)s' cannot be opened for read access. Operation timed out."
msgstr ""

#: src/sbackup/util/filecollect.py:368
#, python-format
msgid ""
"File '%(file)s' cannot be opened for read access with error '%(error)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:385
#, python-format
msgid "File '%(file)s' is backup's target directory."
msgstr ""

#: src/sbackup/util/filecollect.py:394
#, python-format
msgid "Symbolic link '%(path)s' -> '%(ln_target)s' is a circular symlink."
msgstr ""

#: src/sbackup/util/filecollect.py:414
#, python-format
msgid ""
"File '%(file)s' exceeds maximum file size ( %(filesize)s > %(maxsize)s)."
msgstr ""

#: src/sbackup/util/filecollect.py:424
#, python-format
msgid "File '%(file)s' matches regular expression '%(regex)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:430
#, python-format
msgid "Path '%(file)s' defined in excludes list."
msgstr ""

#: src/sbackup/util/filecollect.py:501
#, python-format
msgid "Error while checking directory '%(dir)s': %(error)s."
msgstr ""

#: src/sbackup/util/filecollect.py:552
msgid "Empty regular expression found. Skipped."
msgstr ""

#: src/sbackup/util/filecollect.py:558
#, python-format
msgid "Invalid regular expression ('%s') found. Skipped."
msgstr ""

#: src/sbackup/util/filecollect.py:577
msgid "No directories to backup defined."
msgstr ""

#: src/sbackup/util/lock.py:76
msgid "Another application instance is already running."
msgstr ""

#: src/sbackup/util/lock.py:78
msgid "Invalid lock file found. Is being removed."
msgstr ""

#: src/sbackup/util/lock.py:128 src/sbackup/util/lock.py:142
#, python-format
msgid "Unable to remove lock file: %s"
msgstr ""

#: src/sbackup/util/lock.py:132 src/sbackup/util/lock.py:144
msgid "Unable to remove lock file: File not found."
msgstr ""

#: data/ui/sbackup-config.glade:11
msgid "Exclude file type"
msgstr "Útiloka skáargerð"

#: data/ui/sbackup-config.glade:26
msgid "Standard file types:"
msgstr "Hefðbundin skráarsnið"

#: data/ui/sbackup-config.glade:63
msgid "Custom defined file type with "
msgstr "Skráarsnið sem enda á "

#: data/ui/sbackup-config.glade:93
msgid "file extension"
msgstr "."

#: data/ui/sbackup-config.glade:163
msgid "Exclude by regex"
msgstr "Útiloka með regex"

#: data/ui/sbackup-config.glade:180
msgid ""
"Enter a regular expression.\n"
"Files and directories matching this expression will be excluded from backup."
msgstr ""
"Sláðu inn regex segð.\n"
"Skrár og möppur sem passa við segðina verða útilokaðar frá afritunartökuni."

#: data/ui/sbackup-config.glade:257
msgid "Enter the remote target to include"
msgstr ""

#: data/ui/sbackup-config.glade:278
msgid "Choose a plugin to get the format"
msgstr ""

#: data/ui/sbackup-config.glade:295
msgid "Choose your plugin :"
msgstr ""

#: data/ui/sbackup-config.glade:367
msgid "Check"
msgstr ""

#: data/ui/sbackup-config.glade:479
msgid ""
"<b>Save changed configuration?</b>\n"
"\n"
"The configuration has changed. If you want to use the modified "
"configuration, you have to save it.\n"
"\n"
"Do you want to save the configuration now?"
msgstr ""

#: data/ui/sbackup-config.glade:572
msgid "Configuration Profile Manager"
msgstr ""

#: data/ui/sbackup-config.glade:594
msgid ""
"You can set up multiple configuration profiles which are processed when "
"performing a backup. Each profile has its own set of settings including "
"backup destination, backup format etc."
msgstr ""

#: data/ui/sbackup-config.glade:613
msgid "<b>Configuration profiles</b>"
msgstr ""

#: data/ui/sbackup-config.glade:785
msgid ""
"<b>Add new profile</b>\n"
"\n"
"Please enter the new profile name and set the according checkbox in order to "
"enable resp. disable it."
msgstr ""

#: data/ui/sbackup-config.glade:810
msgid "Profile name"
msgstr ""

#: data/ui/sbackup-config.glade:836
msgid "Enabled"
msgstr ""

#: data/ui/sbackup-config.glade:915 data/ui/sbackup-restore.glade:7
msgid "Simple Backup Suite"
msgstr "Simple Backup Suite"

#: data/ui/sbackup-config.glade:932
msgid "_File"
msgstr ""

#: data/ui/sbackup-config.glade:971
msgid "Save as..."
msgstr ""

#: data/ui/sbackup-config.glade:1008
msgid "_Tools"
msgstr ""

#: data/ui/sbackup-config.glade:1016 data/ui/sbackup-config.glade:1140
msgid "Set default settings..."
msgstr ""

#: data/ui/sbackup-config.glade:1037
msgid "Profile Manager..."
msgstr ""

#: data/ui/sbackup-config.glade:1058
msgid "Make backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1079
msgid "_Help"
msgstr ""

#: data/ui/sbackup-config.glade:1087
msgid "_View all help topics"
msgstr ""

#: data/ui/sbackup-config.glade:1103
msgid "_About this program"
msgstr ""

#: data/ui/sbackup-config.glade:1154
msgid "Reload configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1168
msgid "Save configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1191
msgid "Make a backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1214
msgid "Open the profile manager"
msgstr ""

#: data/ui/sbackup-config.glade:1215
msgid "Profiles"
msgstr ""

#: data/ui/sbackup-config.glade:1253
msgid ""
"none\n"
"gzip\n"
"bzip2"
msgstr ""

#: data/ui/sbackup-config.glade:1271
msgid ""
"Compression format\n"
"Default is set to uncompressed backups."
msgstr ""

#: data/ui/sbackup-config.glade:1290 data/ui/sbackup-config.glade:1403
msgid ""
"You can split the snapshot to make backup on some filesystem that doesn't "
"support large files. This functionality can not be used if the backup is "
"compressed."
msgstr ""

#: data/ui/sbackup-config.glade:1330
msgid "Do a full backup at least once every"
msgstr "Framkvæma heildarafritun á amk."

#: data/ui/sbackup-config.glade:1339
msgid ""
"Unless a set number of days has passed since the last full backup, "
"incremental backups will be done."
msgstr ""

#: data/ui/sbackup-config.glade:1356 data/ui/sbackup-config.glade:3015
msgid "days"
msgstr "daga fresti."

#: data/ui/sbackup-config.glade:1370
msgid " "
msgstr ""

#: data/ui/sbackup-config.glade:1380 data/ui/sbackup-config.glade:2323
msgid "MiB"
msgstr ""

#: data/ui/sbackup-config.glade:1405
msgid "Split backup archives into several chunks"
msgstr ""

#: data/ui/sbackup-config.glade:1438 data/ui/sbackup-config.glade:2627
#: data/ui/sbackup-config.glade:2863 data/ui/sbackup-config.glade:3111
#: data/ui/sbackup-config.glade:3856
msgid "Help"
msgstr ""

#: data/ui/sbackup-config.glade:1442 data/ui/sbackup-config.glade:2632
msgid "Show help topic about configuration GUI."
msgstr ""

#: data/ui/sbackup-config.glade:1475
msgid "General"
msgstr "Almennt"

#: data/ui/sbackup-config.glade:1495
msgid "<b>Included files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1570 data/ui/sbackup-config.glade:1818
#: data/ui/sbackup-config.glade:2023 data/ui/sbackup-config.glade:2181
msgid "Remove"
msgstr ""

#: data/ui/sbackup-config.glade:1620 data/ui/sbackup-config.glade:1868
msgid "Add File"
msgstr "Bæta við skrá"

#: data/ui/sbackup-config.glade:1671 data/ui/sbackup-config.glade:1919
msgid "Add Directory"
msgstr "Bæta við möppu"

#: data/ui/sbackup-config.glade:1718
msgid "Include"
msgstr "Til afritunar"

#: data/ui/sbackup-config.glade:1739
msgid "<b>Excluded files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1952
msgid "Paths"
msgstr "Slóðir"

#: data/ui/sbackup-config.glade:2074 data/ui/sbackup-config.glade:2232
msgid "Add"
msgstr ""

#: data/ui/sbackup-config.glade:2109
msgid "File types"
msgstr "Gerðir"

#: data/ui/sbackup-config.glade:2267
msgid ""
"Regular\n"
"Expressions"
msgstr ""

#: data/ui/sbackup-config.glade:2288
msgid "Do not backup files bigger than"
msgstr ""

#: data/ui/sbackup-config.glade:2344
msgid "Follow symbolic links"
msgstr ""

#: data/ui/sbackup-config.glade:2373
msgid "Others"
msgstr ""

#: data/ui/sbackup-config.glade:2400
msgid "Exclude"
msgstr "Útiloka"

#: data/ui/sbackup-config.glade:2421
msgid "Use default backup directory"
msgstr ""

#: data/ui/sbackup-config.glade:2446
msgid "/var/backup"
msgstr ""

#: data/ui/sbackup-config.glade:2472
msgid "Use custom local backup directory"
msgstr "Nota valda möppu undir afritun"

#: data/ui/sbackup-config.glade:2500
msgid "Select the destination folder"
msgstr "Veldu möppu"

#: data/ui/sbackup-config.glade:2526
msgid "Use a remote site"
msgstr ""

#: data/ui/sbackup-config.glade:2568
msgid "ssh://username:password@example.com/remote/dir/"
msgstr "ssh://notandi:lykilorð@tölvunafn/slóð/á/möppu"

#: data/ui/sbackup-config.glade:2580
msgid "Connect..."
msgstr ""

#: data/ui/sbackup-config.glade:2672
msgid ""
"<i>The selected destination is not writeable with current permissions. "
"Backups cannot be stored there.</i>"
msgstr ""

#: data/ui/sbackup-config.glade:2704
msgid "Destination"
msgstr "Áfangastaður"

#: data/ui/sbackup-config.glade:2725
msgid "_No scheduled backups"
msgstr ""

#: data/ui/sbackup-config.glade:2742
msgid "_Simple"
msgstr ""

#: data/ui/sbackup-config.glade:2747
msgid ""
"This option will make a backup not at a precise time but when the computer "
"is up. "
msgstr ""

#: data/ui/sbackup-config.glade:2761
msgid "_Custom"
msgstr ""

#: data/ui/sbackup-config.glade:2781
msgid "Do backups"
msgstr ""

#: data/ui/sbackup-config.glade:2798
msgid "Cron time definition"
msgstr ""

#: data/ui/sbackup-config.glade:2834
msgid ""
"hourly\n"
"daily\n"
"weekly\n"
"monthly"
msgstr ""

#: data/ui/sbackup-config.glade:2867
msgid "Show help topic about scheduled backups."
msgstr ""

#: data/ui/sbackup-config.glade:2905
msgid ""
"<small>You can configure scheduled (i.e. automated) backups either simply by "
"choosing an interval or precisely by a custom cron definition. See \"man 5 "
"crontab\" for cron time definition format.</small>"
msgstr ""

#: data/ui/sbackup-config.glade:2937
msgid "Schedule"
msgstr ""

#: data/ui/sbackup-config.glade:2955
msgid "Enable purging of old backups"
msgstr ""

#: data/ui/sbackup-config.glade:2983
msgid "Simple cut-off: Erase all backups older than"
msgstr ""

#: data/ui/sbackup-config.glade:3039
msgid "Logarithmic"
msgstr ""

#: data/ui/sbackup-config.glade:3065
msgid ""
"<small>Keep progressively less backups into the past:\n"
"Keep all backups from yesterday\n"
"Keep one backup per day from last week\n"
"Keep one backup per week from last month\n"
"Keep one backup per month from last year\n"
"Keep one backup per quarter from 2nd last year\n"
"Keep one backup per year further into past</small>"
msgstr ""

#: data/ui/sbackup-config.glade:3116
msgid "Show help topic about purging of backups."
msgstr ""

#: data/ui/sbackup-config.glade:3152
msgid ""
"<small>Note: Additionally all incomplete and irrecoverable backups will be "
"erased as long as there is a complete backup snapshot made later on.</small>"
msgstr ""
"Ath: Til viðbótar þá verður öllum ókláruðum og skemmdum afritum hent svo "
"lengi sem það er til nýrri útgáfa."

#: data/ui/sbackup-config.glade:3184
msgid "Purging"
msgstr "Geymsla afrita"

#: data/ui/sbackup-config.glade:3186
msgid "Cleaning old backups"
msgstr "Hreinsa gömul afrit"

#: data/ui/sbackup-config.glade:3209
msgid "<b>Logging</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3234
msgid "Debug is just for debbuging purpose.Info is the recommended setting"
msgstr ""

#: data/ui/sbackup-config.glade:3236
msgid "Log level"
msgstr ""

#: data/ui/sbackup-config.glade:3248
msgid "Debug is just for debbuging purpose. Info is the recommended setting."
msgstr ""

#: data/ui/sbackup-config.glade:3274
msgid "Log file directory"
msgstr ""

#: data/ui/sbackup-config.glade:3320
msgid "<b>Report by mail</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3350
msgid "From"
msgstr ""

#: data/ui/sbackup-config.glade:3361
msgid "SMTP server"
msgstr ""

#: data/ui/sbackup-config.glade:3375 data/ui/sbackup-config.glade:4053
msgid "Port"
msgstr ""

#: data/ui/sbackup-config.glade:3391
msgid "To"
msgstr ""

#: data/ui/sbackup-config.glade:3478
msgid "Use a username and password"
msgstr ""

#: data/ui/sbackup-config.glade:3508 data/ui/sbackup-config.glade:4075
msgid "Username"
msgstr ""

#: data/ui/sbackup-config.glade:3540 data/ui/sbackup-config.glade:4124
msgid "Password"
msgstr ""

#: data/ui/sbackup-config.glade:3584
msgid "Use secure connection"
msgstr ""

#: data/ui/sbackup-config.glade:3615
msgid "Choose a certificate file"
msgstr ""

#: data/ui/sbackup-config.glade:3629
msgid "Key"
msgstr ""

#: data/ui/sbackup-config.glade:3645
msgid "Certificate"
msgstr ""

#: data/ui/sbackup-config.glade:3659
msgid "Choose a key file"
msgstr ""

#: data/ui/sbackup-config.glade:3731
msgid "Test _mail settings"
msgstr ""

#: data/ui/sbackup-config.glade:3772
msgid "Report"
msgstr ""

#: data/ui/sbackup-config.glade:3811
msgid "Warning"
msgstr ""

#: data/ui/sbackup-config.glade:3860
msgid "Show help topic about default values."
msgstr ""

#: data/ui/sbackup-config.glade:3893
msgid "_Set"
msgstr ""

#: data/ui/sbackup-config.glade:3935
msgid "Set remote target"
msgstr ""

#: data/ui/sbackup-config.glade:3957
msgid "Type of service"
msgstr ""

#: data/ui/sbackup-config.glade:3997
msgid "Hostname"
msgstr ""

#: data/ui/sbackup-config.glade:4027
msgid "<b>Details</b>"
msgstr ""

#: data/ui/sbackup-config.glade:4157
msgid "Show password"
msgstr ""

#: data/ui/sbackup-config.glade:4217
msgid "_Connect"
msgstr ""

#: data/ui/sbackup-config.glade:4293
msgid "Make a full backup"
msgstr ""

#: data/ui/sbackup-restore.glade:38
msgid "<b>Backup destination</b>"
msgstr ""

#: data/ui/sbackup-restore.glade:52
msgid "Displays the manual in Help browser."
msgstr ""

#: data/ui/sbackup-restore.glade:85
msgid "Default profile's destination"
msgstr ""

#: data/ui/sbackup-restore.glade:90
msgid ""
"Set backup destination to the place specified in default configuration "
"profile"
msgstr ""

#: data/ui/sbackup-restore.glade:137
msgid "Custom destination"
msgstr ""

#: data/ui/sbackup-restore.glade:143
msgid ""
"Select a backup destination on your harddisk or at a remote site. Press "
"Apply when ready."
msgstr ""

#: data/ui/sbackup-restore.glade:184
msgid "Choose a destination folder. For remote sites fill URI in text field"
msgstr ""

#: data/ui/sbackup-restore.glade:201
msgid "Change destination to path specified in left text field"
msgstr ""

#: data/ui/sbackup-restore.glade:266
msgid "Select a bold printed date to get the list of backups for that date."
msgstr ""

#: data/ui/sbackup-restore.glade:310
msgid "Available backups"
msgstr ""

#: data/ui/sbackup-restore.glade:398
msgid "Delete snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:436
msgid "Delete"
msgstr ""

#: data/ui/sbackup-restore.glade:475
msgid "Upgrade snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:515
msgid "Upgrade"
msgstr ""

#: data/ui/sbackup-restore.glade:553
msgid "<b>Snapshot History</b>"
msgstr ""

#: data/ui/sbackup-restore.glade:579
msgid "Current base:"
msgstr ""

#: data/ui/sbackup-restore.glade:620
msgid "Select a snapshot on this list to rebase on it."
msgstr ""

#: data/ui/sbackup-restore.glade:695
msgid "Rebase"
msgstr ""

#: data/ui/sbackup-restore.glade:746
msgid "<b>Snapshot Management</b>"
msgstr ""

#: data/ui/sbackup-restore.glade:785
msgid "Snapshot details"
msgstr ""

#: data/ui/sbackup-restore.glade:850 data/ui/sbackup-restore.glade:867
msgid "Restores files/directories from the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:862
msgid "Restore"
msgstr "Endurheimta"

#: data/ui/sbackup-restore.glade:882 data/ui/sbackup-restore.glade:901
msgid ""
"Restores files/directories recursively starting from the base snapshot to "
"the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:896
msgid "Revert"
msgstr ""

#: data/ui/sbackup-restore.glade:939
msgid "<b>Restoration Management</b>"
msgstr ""

#: data/ui/sbackup-restore.glade:964
msgid "<b>Export Management</b>"
msgstr ""

#: data/ui/sbackup-restore.glade:1056
msgid "Headline"
msgstr ""

#: data/ui/sbackup-restore.glade:1069
msgid "Content"
msgstr ""
